tests/util/models/gpflow/models.py

```diff
@@ -381,6 +381,16 @@ def svgp_model(x: tf.Tensor, y: tf.Tensor, num_latent_gps: int = 1) -> SVGP:
     )
 
 
+def quadratic_mean_rbf_kernel_model(dataset: Dataset) -> QuadraticMeanAndRBFKernelWithSamplers:
+    model = QuadraticMeanAndRBFKernelWithSamplers(
+        noise_variance=tf.constant(1.0, dtype=tf.float64), dataset=dataset
+    )
+    model.kernel = (
+        gpflow.kernels.RBF()
```

Comment:
maybe use a non=trivial value for the RBF parameters? e.g. variance=0.9

the default variance = 1 makes every test a bit weaker IMO

Reply:
It was 1.0 originally in all the tests, but I'll change it.

Reply:
Changed to 0.9.